src/LeafletWithGlobals.js

```diff
@@ -0,0 +1,22 @@
+import * as L from './Leaflet';
+export * from './Leaflet';
+
+var globalL = L.extend(L, {noConflict: noConflict});
+var globalObject = getGlobalObject();
+var oldL = globalObject.L;
+
+globalObject.L = globalL;
+
+export function noConflict() {
+	globalObject.L = oldL;
+	return globalL;
+}
+
+function getGlobalObject() {
+	if (typeof globalThis !== 'undefined') { return globalThis; }
```

Comment:
What's the purpose of `globalThis`? Framework compatibility?

Reply:
The [`globalThis`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis) provides a standard way of accessing the global `this` value (and hence the global object itself) across environments.

Reply:
I see, it was added to browsers fairly recently and wasn't aware of it.

Reply:
Yes it does, as `globalThis` refers to the global in Node.js as well. But it wont prevent errors that occur if certain APIs don't exist (such as the DOM).